pytorch_lightning/core/lightning.py

```diff
@@ -252,10 +252,15 @@ def truncated_bptt_steps(self, truncated_bptt_steps: int) -> None:
         self._truncated_bptt_steps = truncated_bptt_steps
 
     @property
-    def logger(self):
+    def logger(self) -> Optional[LightningLoggerBase]:
         """Reference to the logger object in the Trainer."""
         return self.trainer.logger if self.trainer else None
 
+    @property
+    def loggers(self) -> Optional[List[LightningLoggerBase]]:
+        """Reference to the loggers object in the Trainer."""
+        return self.trainer.loggers if self.trainer else None
+
```

Comment:
keep the types consistent. this should return an empty list, just like trainer.loggers